video_xblock/utils.py

```diff
@@ -0,0 +1,10 @@
+"""
+This module contains video xblock helpers.
+"""
+
+
+def ugettext(text):
```

Comment:
This was to fix pep8 violation: `E731 do not assign a lambda expression, use a def`